backtrace-library/src/main/java/backtraceio/library/services/BacktraceNativeCrashResponseListener.java

```diff
@@ -0,0 +1,128 @@
+package backtraceio.library.services;
+
+import android.util.Log;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import backtraceio.library.BacktraceClient;
+import backtraceio.library.base.BacktraceBase;
+import backtraceio.library.common.BacktraceConstants;
+import backtraceio.library.events.OnServerResponseEventListener;
+import backtraceio.library.logger.BacktraceLogger;
+import backtraceio.library.models.BacktraceResult;
+import backtraceio.library.models.json.BacktraceReport;
+import backtraceio.library.models.types.BacktraceResultStatus;
+import kotlin.text.Regex;
+
+public class BacktraceNativeCrashResponseListener
+        implements OnServerResponseEventListener {
+    private static final String LOG_TAG = BacktraceNativeCrashResponseListener.
+            class.getSimpleName();
+
+    private static BacktraceNativeCrashResponseListener instance;
+
+    private BacktraceNativeCrashResponseListener() {}
+
+    @Override
+    public void onEvent(BacktraceResult backtraceResult) {
+        if (backtraceResult.status == BacktraceResultStatus.Ok) {
+            // Get the minidump out
+            BacktraceReport report = backtraceResult.getBacktraceReport();
+            if (report != null) {
+                List<String> attachments = report.attachmentPaths;
+                for (String attachment : attachments) {
+                    if (attachment.endsWith(".dmp")) {
+                        moveToCompleted(attachment);
+                        break;
+                    }
+                }
+            }
+        }
+    }
+
+    public static void UploadMinidumps(BacktraceBase client, String databasePath) {
+        if (client == null) {
+            BacktraceLogger.e(LOG_TAG, "Backtrace client can't be null");
+            return;
+        }
+        if (databasePath.equals("")) {
```

Comment:
maybe it will be better to use method `isNullOrEmpty` from `BacktraceStringHelper`